src/Selenium2Library/keywords/browsermanagement.py

```diff
@@ -12,6 +12,9 @@
 from Selenium2Library.locators.windowmanager import WindowManager
 from Selenium2Library.robotlibcore import keyword
 
+from robot.api import logger
```

Comment:
Robot logger is not used anymore directly, if you need logging they are found from self.